fighthealthinsurance/models.py

```
@@ -340,12 +342,24 @@ class FaxesToSend(ExportModelOperationsMixin("FaxesToSend"), models.Model):  # t
+            try:
+                return cryptographer.decrypt(self.combined_document_enc.read())
+            except:
+                logger.opt(exception=True).debug(
+                    f"Error reading encrypted document, sometimes this mean it was not encrypted falling back"
+                )
+                self.combined_document_enc.read()
+
     def get_temporary_document_path(self):
-        combined_document = self.combined_document or self.combined_document_enc
         with tempfile.NamedTemporaryFile(
             suffix=combined_document.name, mode="w+b", delete=False
```

Ruff (0.8.2)

`combined_document` may be undefined, or defined from star imports

(F405)